### ISSUE TITLE
Low Order Building Model collective issue

### ISSUE THREAD

**Issue by DaJansenGit** [RWTH-EBC/AixLib#770]
I want to collect all the ongoing issues for AixLib and TEASER due to the ongoing work for the internal gains changes, adding moisture and renaming multizone model at one place to have a better overview over the progress 

## AixLib
* [x] Moisture Balance: #695 
* [x] Renaming multizone model: #707 
* [x] Adjustments in AixLib due to internal gains changes: #690
* [x] Threshold for ideal heater - merge #823 and https://github.com/RWTH-EBC/TEASER/pull/618
* [x] Fix typo: AixLib.Utilities.Sources.InternalGains.Humans.HumanTotalHeat_TemperatureDepdendent

## TEASER
* [x] Internal gains changes (moisture and thermal):  https://github.com/RWTH-EBC/TEASER/issues/567

**Comment by DaJansenGit** [comment]
To finish this topic I would suggest the following way:
1. wait until https://github.com/ibpsa/modelica-ibpsa/issues/1209 is done
2. create new branch for this issue #770 from development after #774 is done
3. merge all above listed branches into #770 
4. merge #770 into development
5. make a new release :)

you agree @KremerMartin

**Comment by KremerMartin** [comment]
@DaJansenGit I agree to that. I will do the pull request on [ibpsa #1209](https://github.com/ibpsa/modelica-ibpsa/issues/1209l) today.

**Comment by KremerMartin** [comment]
The changes made on the ROM were merged in the IBPSA sub-branch [issue1209_thermalMoistAir](https://github.com/ibpsa/modelica-ibpsa/tree/issue1209_thermalZoneMoistAir).
So we will need to wait until the changes are merged into the master of IBPSA to avoid merge conflicts updating our master.

**Comment by Dawac** [comment]
Hi @KremerMartin and @DaJansenGit, how did this issue progress so far? Do you think this could be solved at our upcoming Hackday?

**Comment by DaJansenGit** [comment]
Yes, it's our plan to close this issue during the hackday ;)

**Comment by DaJansenGit** [comment]
this is closed with dc57456
